tests/sentry/web/frontend/test_auth_channel_login.py

```diff
@@ -1,22 +1,26 @@
-from functools import cached_property
-
 from django.urls import reverse
 
+from sentry.auth.providers.fly.provider import FlyOAuth2Provider
+from sentry.models.authprovider import AuthProvider
 from sentry.testutils.cases import TestCase
 from sentry.testutils.silo import region_silo_test
 
 
 @region_silo_test
-class AuthOrganizationIdentifierLoginTest(TestCase):
-    @cached_property
-    def organization(self):
-        return self.create_organization(name="test org", owner=self.user)
-
-    @cached_property
-    def path(self):
-        return reverse("sentry-auth-organization-id", args=[self.organization.id])
+class AuthOrganizationChannelLoginTest(TestCase):
+    def setup(self):
+        self.organization = self.create_organization(name="test org", owner=self.user)
+        self.partner_org_id = "foobar"
+        config_data = FlyOAuth2Provider.build_config(resource={"id": self.partner_org_id})
+        self.auth_provider = AuthProvider.objects.create(
+            organization_id=self.organization.id, provider="fly", config=config_data
+        )
```

Comment:
mypy is complaining cuz some parent of `TestCase` has an [`auth_provider` function](https://github.com/getsentry/sentry/blob/master/src/sentry/testutils/cases.py#L217). Prob fine to ignore or just rename the var

Reply:
oof thanks 😕